README.md

```diff
@@ -29,3 +29,11 @@ protoc --doc_out="template=templates/tmpl.html:doc/" file.proto
 ```
 
 Would produce documentation using `templates/tmpl.html` inside the `doc/` output directory for `file.proto`.
+
+# proto_path issues
+
+You will quickly find that the generator fails if you run it with a `--proto_path` argument. This is because `protoc` does not pass it's command-line arguments onto generator programs (and `protoc-gen-doc` needs to know the `--proto_path` arguments in order to resolve symbols appropriately). To workaround this issue you must export the `$PROTO_PATH` environment variable, thus you can run `protoc` as just:
```

Comment:
Regarding the actual content, I find this very surprising. If the content of `-I` flag (aka `--proto_path` flag) is not given to the generator, how do all other generators like `protoc-gen-go`, `protoc-gen-gogo`, `protoc-gen-grpc-gateway` find the imports successfully?

Or does `protoc` itself deal with imports and provides the relevant info to the plugin binary.

I don't know how it works, but I found it surprising, that's why I wanted to ask.